### PR TITLE
Updating the correct Authority URL for Azure Gov

### PR DESCRIPTION
**Pull request by zakramer**
Authority URL going forward is http://login.microsoftonline.us.

@bernie-msft for awareness.

### CODE REVIEW

**Comment by zakramer** [comment]
Please ping Bernarldo Ellis internally if there are any questions.

File: sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Beginning with version 1702, you can use the OMS connector to connect to an OMS
 
 ### Install the OMS Connector  
 1. In the Configuration Manager console, configure your [hierarchy to use pre-release features](/sccm/core/servers/manage/pre-release-features), and then enable the use of the OMS Connector.  
-
+0
```
Comment (mattbriggs) [inline sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md]: Please remove the stray "0".

File: sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md [hunk 2]
```diff
@@ -132,15 +132,15 @@ After you have linked Configuration Manager to OMS, you can add or remove collec
...
+  Change the value for the setting name *FairFaxAuthorityResource* to be equal to "https://login.microsoftonline.us/"
 
   - **Original:**
     &lt;setting name="FairFaxAuthorityResource" serializeAs="String">   
     &lt;value>&lt;/value>
 
 	- **Edited:**
     &lt;setting name="FairFaxAuthorityResource" serializeAs="String">   
-    &lt;value>https://login.microsoftonline.com/&lt;/value>
+    &lt;value>https://login.microsoftonline.us/&lt;/value>
```
Comment (mattbriggs) [inline sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md]: Verified. I can accept with the other revision. Thanks!

Comment (arob98) [reply sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md]: @mattbriggs want me to merge?

Comment (mattbriggs) [reply sccm/core/clients/manage/sync-data-microsoft-operations-management-suite.md]: @arob98 sure. Thanks! Matt

**Pull request merged**